source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/UtilityFunctions/SeriesPredictedValueUtilityFunction.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+import numpy as np
+import pandas as pd
+from scipy.stats import t
+from mlos.Logger import create_logger
+from mlos.Optimizers.OptimizationProblem import SeriesObjective
+from mlos.Optimizers.ExperimentDesigner.UtilityFunctions.UtilityFunction import UtilityFunction
+from mlos.Optimizers.RegressionModels.MultiObjectiveRegressionModel import MultiObjectiveRegressionModel
+from mlos.Optimizers.RegressionModels.Prediction import Prediction
+from mlos.Spaces import SimpleHypergrid, ContinuousDimension, CategoricalDimension, Point
+from mlos.Spaces.Configs.ComponentConfigStore import ComponentConfigStore
+from mlos.Tracer import trace
+
+
+class SeriesPredictedValueUtilityFunction(UtilityFunction):

```

Comment:
What is the point of this class?

Reply:
To copy the PredictedValueUtilityFunction class. When we are finding the optimum, we don't want anything to do with probability of improvement, we just want the basic result. 

Thoughts on a way to reduce code duplication?